apps/site/src/app/admin/applicants/[uid]/components/ApplicantOverview.tsx

```diff
@@ -0,0 +1,41 @@
+import Box from "@cloudscape-design/components/box";
+import ColumnLayout from "@cloudscape-design/components/column-layout";
+import Container from "@cloudscape-design/components/container";
+import Header from "@cloudscape-design/components/header";
+
+import ApplicantStatus from "@/app/admin/applicants/components/ApplicantStatus";
+import { Applicant } from "@/lib/admin/useApplicant";
+
+import ApplicationReviews from "./ApplicationReviews";
+
+interface ApplicantOverviewProps {
+	applicant: Applicant;
+}
+
+function ApplicantOverview({ applicant }: ApplicantOverviewProps) {
+	const { application_data, status } = applicant;
+	const { submission_time, reviews } = application_data;
+
+	const submittedDate = new Date(submission_time).toDateString();
```

Comment:
Note for later: Might be good to include the time of submission instead of just the date.